data/lang/core/es.json

```diff
@@ -1105,7 +1105,7 @@
   },
   "OVER_N_BILLION": {
     "description": "",
-    "message": "Alrededor de %{population}000 millones"
+    "message": "Alrededor de %%population000 millones"
```

Comment:
This will need to be manually fixed.

Reply:
Yeah, I spotted that one, and I initially assumed the script was doing something wonky, but it is actually doing what it is supposed to:
```
2024-07-03 14:19:40.474 | DEBUG    | __main__:compare_file:127 - translated: OVER_N_BILLION, ['{population}']
2024-07-03 14:19:40.474 | DEBUG    | __main__:compare_file:128 - correct:    OVER_N_BILLION, ['%population']
2024-07-03 14:19:40.474 | DEBUG    | __main__:fix_single_typo:96 - guess: %population, wrong: {population}
2024-07-03 14:19:40.474 | DEBUG    | __main__:fix_single_typo:98 - suggested: ['%population']
2024-07-03 14:19:40.501 | INFO     | __main__:fix_file:215 - old: Alrededor de %{population}000 millones
2024-07-03 14:19:40.501 | INFO     | __main__:fix_file:217 - new: Alrededor de %%population000 millones
```

My plan is to just open an issue on the strings that are "beyond broken" in transifex, and hope translator fixes it.